react-app/src/pages/BulkUpload.tsx

```diff
@@ -214,16 +215,21 @@ const BulkUpload = () => {
               <ListItemText primary={'Optional Headers may have blank values.'} />
               <List sx={{ listStyle: 'disc', pl: 4 }}>
                 {[
-                  'LandLegalDescription',
                   'Description',
                   'Address',
                   'PIN',
                   'Assessed',
                   'AssessedYear',
                   'Netbook',
                   'FiscalYear',
-                  'RentableArea (for Buildings)',
+                  'IsSensitive',
+                  'isVisibleToOtherAgencies',
```

Comment:
Should we capitalize this to match the other headers?
Looks like it's expecting capitalization in the backend as well.